src/main/java/mate/academy/spring/mapper/impl/request/UserRequestMapper.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.spring.mapper.impl.request;
+
+import mate.academy.spring.mapper.DtoRequestMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserRequestMapper
```

Comment:
Seems like you don't use UserRequestMapper class anywhere.

Reply:
true